schemas/com.snowplowanalytics.snowplow/cross_navigation/jsonschema/1-0-0

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a user and session identifiers entity included in cross platform or cross domain navigation.",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "cross_navigation",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "user_id": {
+            "description": "The business user ID, the user was identified in the source app.",
+            "type": [
+                "null",
+                "string"
+            ],
+            "maxLength": 4096
+        },
+        "domain_user_id": {
+            "type": "string",
+            "maxLength": 36,
```

Comment:
The maxLength needs to be at least 128 ([as is the `refr_domain_userid`](https://github.com/snowplow/iglu-central/blob/0d08c92ebde14b77c2f191b64622ac088c70f230/schemas/com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0#L449)).